rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/license/Sw360LicenseService.java

```diff
@@ -161,4 +169,24 @@ public void importSpdxInformation(User sw360User) throws TException {
             throw new HttpMessageNotReadableException("Unable to import All Spdx license. User is not admin");
         }
     }
+
+    public void uploadLicense(User sw360User, MultipartFile file, HttpServletRequest request, boolean overwriteIfExternalIdMatches, boolean overwriteIfIdMatchesEvenWithoutExternalIdMatch) throws IOException, TException {
```

Comment:
remove unused param **HttpServletRequest request** and HttpServletRequest import

Reply:
done.